pyalgomate/core/position.py

```diff
@@ -1,10 +1,11 @@
+from pyalgotrade.strategy.position import WaitingEntryState, OpenState, ClosedState, Position
```

Comment:
Can we move this when the DB changes are done as this change is not going to affect the current working in any way?